x-pack/platform/plugins/shared/alerting/common/routes/maintenance_window/shared/constants/v1.ts

```diff
@@ -14,3 +14,12 @@ export const maintenanceWindowCategoryIdTypes = {
 
 export type MaintenanceWindowCategoryIdTypes =
   (typeof maintenanceWindowCategoryIdTypes)[keyof typeof maintenanceWindowCategoryIdTypes];
+
+export const maxMaintenanceWindowDocs = 10000;
+
+export const maintenanceWindowStatus = {
```

Comment:
I was going to ask why not use a `const enum`, but I see we are banning them through ESLint rules (I couldn’t find the reason for the ban though)